packages/react-components/react-nav-preview/src/components/NavDrawer/NavDrawer.types.ts

```diff
@@ -1,13 +1,13 @@
-import { InlineDrawerProps, InlineDrawerState } from '@fluentui/react-drawer';
+import { DrawerProps, DrawerState } from '@fluentui/react-drawer';
 import { NavProps } from '../Nav/Nav.types';
 import { NavContextValue } from '../NavContext.types';
 
 /**
  * NavDrawer Props
  */
-export type NavDrawerProps = InlineDrawerProps & NavProps;
+export type NavDrawerProps = DrawerProps & NavProps;
 
 /**
  * State used in rendering NavDrawer
  */
-export type NavDrawerState = InlineDrawerState & NavContextValue;
+export type NavDrawerState = DrawerState & NavContextValue & { type: 'inline' | 'overlay' };
```

Comment:
this feels a little yucky and loose. I'd prefer it if drawer exported the type for me to import.